eng/pipelines/templates/CrossFramework_Tests_On_Windows.yml

```diff
@@ -7,6 +7,11 @@ steps:
       Write-Host "##vso[build.updatebuildnumber]$env:FullVstsBuildNumber"
       Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
+- task: PowerShell@1
+  displayName: "Disable strong name verification"
+  inputs:
+    scriptName: "$(Build.Repository.LocalPath)\\scripts\\utils\\DisableStrongNameVerification.ps1"
```

Comment:
May I ask why we need this action?

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/assembly/delay-sign

see step 4.

Reply:
Thank you.
Was this already disabled on  Microbuild images? I had similar issue with my local VM. Just wondering when why this was required for VM, but not for my local host machine itself.

Reply:
This is probably why you don't remember dealing with this on your local devbox: https://github.com/NuGet/NuGet.Client/blob/927e225f1bbefd3e4df3b5cc3c9371d6f65b723c/build.ps1#L63

And I don't know how MicroBuild's images are configured, but given that it works, I assume that they do it.